packages/generate/src/syntax/toEdgeQL.ts

```diff
@@ -1448,13 +1448,29 @@ const numericalTypes: Record<string, boolean> = {
   "std::float64": true,
 };
 
+function makeLabel(stringified: string): string {
+  const MAX_ITERATIONS = 100;
+  let label = `jsonliteral`;
+  let counter = 0;
+  while (stringified.includes(`$${label}$`) && counter < MAX_ITERATIONS) {
+    label = `${label}${counter}`;
+    counter++;
+  }
+  if (counter === MAX_ITERATIONS) {
```

Comment:
just easier to read / reason

```suggestion
  if (counter >= MAX_ITERATIONS) {
```